### ISSUE TITLE
Remove MangoPay References

### ISSUE THREAD

**Issue by blrhc** [liberapay/liberapay.com#1401]
E.g. here (https://liberapay.com/about/legal). From my understanding they are no longer working with Liberapay.

**Comment by trebmuh** [comment]
I can't see any mention of MangoPay in this page.

**Comment by LiamDawe** [comment]
The English translation has it:

> 11. Cookies

The user acknowledges that cookies are necessary for the service to function and agrees to their use.
Terms of Use of the MangoPay Electronic Money

MangoPay is our payments partner, you have to accept their terms of service to be able to fully use Liberapay.

    Mangopay_Terms-AT.pdf
    Mangopay_Terms-DE.pdf
    Mangopay_Terms-EN.pdf
    Mangopay_Terms-ES.pdf
    Mangopay_Terms-FR.pdf
    Mangopay_Terms-IT.pdf
    Mangopay_Terms-NL.pdf

Privacy Policy

We do our best to protect everyone's privacy: we do not attempt to track people who visit our website, we strive to collect only the personal information we actually need, and we don't sell it to anyone.

**Comment by Changaco** [comment]
The section about MangoPay is only visible if you're logged in and you do have a MangoPay account.

**Comment by benhc123** [comment]
![mangopay](https://user-images.githubusercontent.com/4144334/51784771-f5189600-2145-11e9-9bc2-3261ad78e159.jpeg)

**Comment by Changaco** [comment]
A MangoPay account was created when you filled the identity form: https://liberapay.com/about/me/identity

#1393 was the first step towards storing identity information in our own database instead of in MangoPay accounts.

**Comment by benhc123** [comment]
Ah ok, understood.

**Comment by trebmuh** [comment]
Thanks @Changaco I can see it now that I'm logged in.
What wan we do while still having some money in some Mangopay users accounts?

Leaving it like the way it is seems to indicate to the user that Mangopay is our only payment operator. It is a bit confusing from an user POV. I'd suggest to reword it for the time being.


I suggest rewording like from:
> MangoPay is our payments partner, you have to accept their terms of service to be able to fully use Liberapay.
to
> MangoPay was our payments partner before **DATE**, you had to accept their terms of service to be able to fully use Liberapay if you have started to use Liberapay before **DATE**

Does that make sense to you?

@Changaco @mattbk

**Comment by mattbk** [comment]
@Changaco, do we have any power to close Mangopay accounts, especially for users who have emptied their wallets? 

This seems like a weird limbo to be in. I'm sure there are users out there who don't want an inaccessible Mangopay account in their name. https://support.mangopay.com/s/article/Can-I-request-for-user-data-to-be-deleted?language=en_US

**Comment by Changaco** [comment]
Mangopay accounts don't have an open/closed status, at least not in the API. Once we've copied the relevant identity data into our own database we could ask Mangopay to close/deactivate/delete all non empty accounts.

@trebmuh Feel free to open a pull request to reword the paragraph about Mangopay. Here is my suggestion:

> MangoPay was Liberapay's initial payment partner, but they quit on us in 2018. Below are copies of the contract you had to accept in order to send or receive money through Liberapay prior to August 2018:

**Comment by trebmuh** [comment]
Done here: #1440
